File: DotNet/CesiumLanguageWriter/Advanced/ICesiumStripeOrientationValuePropertyWriter.cs
namespace CesiumLanguageWriter.Advanced
{
    /// <summary>
    /// A writer that can write a value as a stripe orientation.
    /// </summary>
    public interface ICesiumStripeOrientationValuePropertyWriter : ICesiumPropertyWriter
    {
        /// <summary>
        /// Writes the value as a stripe orientation.
        /// </summary>
        /// <param name="value">The orientation.</param>
        void WriteStripeOrientation(CesiumStripeOrientation value);
    }
}

File: Java/CesiumLanguageWriter/translatedSrc/cesiumlanguagewriter/advanced/ICesiumStripeOrientationValuePropertyWriter.java
package cesiumlanguagewriter.advanced;


import agi.foundation.compatibility.*;
import cesiumlanguagewriter.*;
import javax.annotation.Nonnull;

/**
 * A writer that can write a value as a stripe orientation.
 */
@SuppressWarnings( {
        "unused",
        "deprecation",
        "serial"
})
public interface ICesiumStripeOrientationValuePropertyWriter extends ICesiumPropertyWriter {
    /**
    * Writes the value as a stripe orientation.
    * @param value The orientation.
    */
    void writeStripeOrientation(@Nonnull CesiumStripeOrientation value);
}